core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3368,29 +3368,29 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_METADATA_CACHE_ENABLED =
-      new Builder(Name.USER_METADATA_CACHE_ENABLED)
-          .setDefaultValue(false)
-          .setDescription("If this is enabled, metadata of paths will be cached. "
-              + "The cached metadata will be evicted when it expires after "
-              + Name.USER_METADATA_CACHE_EXPIRATION_TIME
-              + " or the cache size is over the limit of "
-              + Name.USER_METADATA_CACHE_MAX_SIZE + ".")
+  public static final PropertyKey USER_FILESYSTEM_CLASS =
```

Comment:
nit: can you put this property in alphabet order?

Reply:
updated, thanks. There are other keys which are not in order, but I won't address that in this PR.